sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -903,6 +903,13 @@ object SQLConf {
     .intConf
     .createWithDefault(4096)
 
+  val PARQUET_COLUMN_INDEX_ACCESS = buildConf("spark.sql.parquet.columnIndexAccess")
```

Comment:
Column index is a special feature in Parquet and this may cause confusion. How about `spark.sql.parquet.accessByIndex`?

Reply:
make sense

Reply:
How about using `ordinal` instead of `index`?